suite/tests/client-interface/low_on_memory.c

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
My prior comment about not being needed was the particular copyright line for years ago for VMware since this was not copied from an old file -- we do want the BSD license header and some sort of copyright line.

Reply:
Done